Source/Scene/Cesium3DTileset.js

```diff
@@ -1316,17 +1316,18 @@ Object.defineProperties(Cesium3DTileset.prototype, {
   },
 
   /**
-   * A custom shader to apply to the tileset. Only used for contents that use
-   * {@link ModelExperimental}. Using custom shaders with a {@link Cesium3DTileStyle}
-   * may lead to undefined behavior.
+   * A custom shader to apply to all tiles in the tileset. Only used for
+   * contents that use {@link ModelExperimental}. Using custom shaders with a
+   * {@link Cesium3DTileStyle} may lead to undefined behavior. Any primitive
+   * that is incompatible with the custom shader code (e.g. due to missing
+   * vertex attributes) will be rendered as if no custom shader were applied.
```

Comment:
> Any primitive that is incompatible with the custom shader code (e.g. due to missing vertex attributes) will be rendered as if no custom shader were applied

Is this true? I thought default attribute values were used.

Reply:
Ah right the default values... probably best if we omit the implementation details, it's a bit complex